src/xenia/base/memory_generic.cc

```diff
@@ -43,7 +43,14 @@ void copy_and_swap_16_aligned(uint16_t* dest, const uint16_t* src,
 
 void copy_and_swap_16_unaligned(uint16_t* dest, const uint16_t* src,
                                 size_t count) {
-  for (size_t i = 0; i < count; ++i) {
+  size_t i;
+  for (i = 0; i + 8 <= count; i += 8) {
+    __m128i s = _mm_loadu_si128((__m128i*)&src[i]);
```

Comment:
Can you make sure these are emitting v prefixed AVX variants? (debug and look at disasm)
I'm not actually sure how VC++ handles that (I believe we are compiling with AVX mode, so it *should* be v prefixed).